src/Raven.Studio/typescript/viewmodels/database/timeSeries/deleteTimeSeries.ts

```diff
@@ -34,11 +35,21 @@ class deleteTimeSeries extends dialogViewModelBase {
         criteria.selection = criteria.selection || [];
         
         this.startDateToUse = ko.pureComputed(() => {
```

Comment:
where do you use that?

Reply:
it is used here:
![image](https://user-images.githubusercontent.com/17376974/144996999-2c7d707a-03a5-4eb7-b156-76adb3e258e2.png)